peer/brontide.go

```diff
@@ -3737,26 +3738,18 @@ func (p *Brontide) addActiveChannel(c *channeldb.OpenChannel) error {
 			err)
 	}
 
-	// We'll query the localChanCfg of the new channel to determine the
-	// minimum HTLC value that can be forwarded. For the maximum HTLC value
-	// that can be forwarded and fees we'll use the default values, as they
-	// currently are always set to the default values at initial channel
-	// creation. Note that the maximum HTLC value defaults to the cap on
-	// the total value of outstanding HTLCs.
-	fwdMinHtlc := lnChan.FwdMinHtlc()
-	defaultPolicy := p.cfg.RoutingPolicy
-	forwardingPolicy := &htlcswitch.ForwardingPolicy{
-		MinHTLCOut:    fwdMinHtlc,
-		MaxHTLC:       c.LocalChanCfg.MaxPendingAmount,
-		BaseFee:       defaultPolicy.BaseFee,
-		FeeRate:       defaultPolicy.FeeRate,
-		TimeLockDelta: defaultPolicy.TimeLockDelta,
+	// We'll query the channel DB for the new channel's initial forwarding
+	// policies to determine the policy we start out with.
+	initialPolicy, err := p.cfg.ChannelDB.GetInitialForwardingPolicy(chanID)
```

Comment:
hmmm got the following errors from the neutrino build,
```
2023-07-24 14:48:33.318 [ERR] PEER: Peer(02d733449d425c564e1382347b67c0adf1a9a2c80f6ea68065d9a0271984f8eecd): unable to query for initial forwarding policy: channel not found
2023-07-24 14:48:33.318 [ERR] FNDG: Unable to add new channel 79e0a85a00632862fb0327cb0cbc5c2f125c8cdb995cd907590b0b850776ee27:0 with peer 02d733449d425c564e1382347b67c0adf1a9a2c80f6ea68065d9a0271984f8eecd: unable to query for initial forwarding policy: channel not found
```
which is very weird because I think we only call `AddNewChannel` in funding manager after the policy is saved to db.